Source/WebCore/dom/Document.cpp

```diff
@@ -9839,6 +9845,13 @@ void Document::didAssociateFormControlsTimerFired()
     if (RefPtr page = this->page(); page && !controls.isEmpty()) {
         ASSERT(m_frame);
         page->chrome().client().didAssociateFormControls(controls, protectedFrame().releaseNonNull());
+
+        Ref event = Event::create(eventNames().webkitassociateformcontrolsEvent, Event::CanBubble::No, Event::IsCancelable::No);
+        event->setIsAutofillEvent();
+        // Do we need to put all the controls into the event somehow?
+        // Should we make a new Event idl and class for this list of elements?
```

Comment:
We most definitely don't want to dispatch an event with a list of elements.
That's just generally not how an event works.